src/uu/chroot/src/chroot.rs

```diff
@@ -94,22 +106,20 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         );
     }
 
-    let command: Vec<&str> = match matches.args.len() {
+    let commands = match matches.values_of(options::COMMAND) {
+        Some(v) => v.map(|v| v).collect(),
```

Comment:
```suggestion
        Some(v) => v.collect(),
```
Unless I'm missing something, I think the map is not necessary.